src/client/main.js

```diff
@@ -27,8 +27,10 @@ export function define(cell) {
   cellsById.get(id)?.variables.forEach((v) => v.delete());
   cellsById.set(id, {cell, variables});
   const root = document.querySelector(`#cell-${id}`);
-  const rejected = (error) => (clear(root), console.error(error), root.append(inspectError(error)));
-  const v = main.variable({_node: root, rejected}, {shadow: {}}); // _node for visibility promise
+  const loading = root.classList.contains("observablehq--loading");
```

Comment:
This is set here for expression code blocks:

https://github.com/observablehq/framework/blob/5b78e40b9e112476775f4781e17510e9608902e3/src/markdown.ts#L111

And here for inline expressions:

https://github.com/observablehq/framework/blob/5b78e40b9e112476775f4781e17510e9608902e3/src/markdown.ts#L265

I suppose we could expose this more explicitly (like we do for the `inline` property), whether there’s an implicit display and hence whether we should show a loading indicator. But it seemed reasonable to just preserve whatever was there before when clearing the error.